lib/ansible/plugins/callback/default.py

```diff
@@ -268,6 +268,22 @@ def v2_playbook_on_stats(self, stats):
 
         self._display.display("", screen_only=True)
 
+        # print custom stats
+        if C.SHOW_CUSTOM_STATS and stats.custom:
+            self._display.banner("CUSTOM STATS: ")
```

Comment:
Splitting this block (273-286) into it's own method would make it easier for sub classes of the default CallbackModule (like debug, skippy, actionable, dense) to extend this behavior.